### PR TITLE
add preview to the readme file

### PR DESCRIPTION
**Pull request by moderakh**
add preview text the readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
-# Go Databricks SQL Driver
+# Preview Go Databricks SQL Driver
 
 Databricks SQL driver for Go's [database/sql](https://golang.org/pkg/database/sql) package.
 
+**NOTE: This Driver is in Preview mode.**
```
Comment (arikfr) [inline README.md:5]: I'm not sure "Preview Mode" will be clear to people? Even if they are familiar with the [Databricks definitions](https://docs.databricks.com/release-notes/release-types.html) - we don't really map to any of the preview phases...

Maybe we should call it beta?

Something like: "Note: this driver is in beta."?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
-# Go Databricks SQL Driver
+# Preview Go Databricks SQL Driver
```
Comment (arikfr) [inline README.md:1]: We should put the label at the end of the title.

Comment (moderakh) [reply README.md]: changed to beta and moved to the end.

File: README.md (hunk 1 above)
Comment (moderakh) [reply README.md]: changed to beta.

**Review by arikfr — APPROVED** [review]

**Pull request merged**